src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
But as you can see, 'uap-aot' doesn't run in app-container.  So, that is why I also have " | IsNetNative" in the "IsUap" definition.

You cannot change 'IsUap" definition untl the uap-aot runs are fixed so as to actually run in app-container.